api-reference/beta/api/subscription_post_subscriptions.md

```diff
@@ -2,7 +2,20 @@
 
 Subscribes a listener application to receive notifications when data on the Microsoft Graph changes.
 ## Prerequisites
-One of the following **scopes**, depending on the target resource, are required to execute this API: *Mail.Read*, *Calendars.Read*, *Contacts.Read*, *Group.Read.All*, *Files.ReadWrite* or *Files.ReadWrite.All*. ***Note:*** The /beta endpoint allows Application permissions for most resources. Conversations in a Group and OneDrive drive root items are not supported with Application permissions.
+One of the following **scopes**, depending on the target resource, are required to execute this API. Creating a subscription requires read scope to the resource. For example, to get notifications messages, your app needs the `Mail.Read` permission. The following table lists the suggested roles needed for each resource.
+
+|Resource/Item |Scope               |
+|--------------|--------------------|
+| Messages     | Mail.Read          |
+| Calendars    | Calendars.Read     |
+| Contacts     | Contacts.Read      |
+| Files        | Files.ReadWrite    |
+| Users        | User.Read.All      |
```

Comment:
Last I remember, this permission scope was not sufficient to read users. Will need to check with Jeff/Dan